### ISSUE TITLE
bug: component explorer code samples hard to read in high contrast mode

### ISSUE THREAD

**Issue by scomea** [microsoft/fast#3980]
In high contrast mode (Windows, Edge) the code samples do not have a high enough contrast ratio.

![image](https://user-images.githubusercontent.com/7649425/94467123-58f32d80-0177-11eb-940c-5300dd247327.png)

**What are the steps to reproduce the issue?**
- Go to https://explore.fast.design/components/fast-accordion
- turn on high contrast light mode
- note code sample in the lower panel

**What behavior did you expect?**
Readable in high-contrast mode with appropriate contrast ratios.

**Comment by janechu** [comment]
@khamudom is there a way to detect when we go to high contrast? Then the solution can be flipping the theme to `hc-black`.

**Comment by khamudom** [comment]
> @khamudom is there a way to detect when we go to high contrast? Then the solution can be flipping the theme to `hc-black`.
> 
> Reference: [microsoft/monaco-editor#472](https://github.com/microsoft/monaco-editor/issues/472)

I can investigate that.

**Comment by janechu** [comment]
This was found to be an issue in the `vs-dark` theme in vscode, an issue has been created for it https://github.com/microsoft/vscode/issues/109402 as well as a pull request https://github.com/microsoft/vscode/pull/109403
